.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!-- Thank you for contributing! -->
+<!-- Please fill this template before submitting your PR (fill the boxes using "x") -->
+
+**Your checklist for this pull request**
+- [ ] I've read the [contributing guideline](CONTRIBUTING.md).
+- [ ] I've tested my changes by building and running the project, and testing changed functionality (if applicable)
+- [ ] I've added automated tests for my change (if applicable, optional)
+- [ ] I've updated documentation to reflect my change (if applicable)
+
+**What is the current behaviour?**
+<!-- Explain how the code works currently -->
+
+**What is the new behaviour?**
+<!-- Explain how the code works after your changes -->
+
+**Test plan**
+<!-- Explain how to test your changes -->
+
+<!-- After submitting, your code will be tested by the CI pipeline. Please
+ensure that all tests pass. If they don't at first, please update your code -->
+
+**Closing issues**
+
+<!-- Add in issue numbers related to this PR, if applicable -->
+
+fixes #issuenumber
```

Comment:
I think that `closes` is better than `fixes`, especially when implementing a feature request, instead of fixing a bug